projects/igniteui-angular/src/lib/badge/badge.component.ts

```diff
@@ -119,6 +119,15 @@ export class IgxBadgeComponent {
     @HostBinding('attr.role')
     public role = 'status';
 
+    /**
+     * The default css class applied to the badge component.
+     *
+     * @hidden
+     * @internal
+     */
+    @HostBinding('class.igx-badge__circle')
+    public styleClass = true;
```

Comment:
```suggestion
    public cssClass = 'igx-badge';
```

Or add this to `setClasses` method as the first member of the `classes` object.